identity-tools/developer_identity/create-dev-identity.sh

```diff
@@ -73,6 +73,9 @@ cli_help() {
 
 OPTIND=1
 
+HW_VERSION="unknown"
+[ -f /proc/device-tree/model ] && HW_VERSION=$(sed 's/ /_/g' <<< $(tr -d '\0' </proc/device-tree/model))
```

Comment:
```suggestion
[ -f /proc/device-tree/model ] && HW_VERSION=$(sed 's/ /_/g' <<< $(tr -d '\0' </proc/device-tree/model)) || true
```

I think this needs to be appended in the off case that the file doesn't exist, since `-e` is set.

Reply:
I tested without `||` on the gateway and looks like `if` the condition is false then its not an error so its just moves on with the default `HW_VERSION` value.